game-core/src/state/game.rs

```diff
@@ -0,0 +1,86 @@
+use amethyst::prelude::*;
+use component::{Enemy, Player};
+use crate::load;
+
+pub struct Game;
+
+impl<'a, 'b> SimpleState<'a, 'b> for Game {
+    fn on_start(&mut self, data: StateData<GameData>) {
+        let world = data.world;
+        let circle_sprite_sheet_handle =
+            load::sprite_sheet(world, "Circle_Spritesheet.png", "Circle_Spritesheet.ron");
+        let background_sprite_sheet_handle =
+            load::sprite_sheet(world, "Background.png", "Background.ron");
+
+        crate::map::load_map_sprites(world);
+        let _background = init::background_sprite(world, &background_sprite_sheet_handle);
+        let _reference = init::reference_sprite(world, &circle_sprite_sheet_handle);
+        let parent = Player::new(world, &circle_sprite_sheet_handle);
+        let _enemy = Enemy::new(world, &circle_sprite_sheet_handle);
+        init::camera(world, parent);
+    }
+}
+
+mod init {
+    use amethyst::utils::ortho_camera::CameraNormalizeMode;
+    use amethyst::utils::ortho_camera::CameraOrtho;
+    use amethyst::{
+        core::{Parent, Transform},
+        ecs::Entity,
+        prelude::*,
+        renderer::{Camera, SpriteRender, SpriteSheetHandle, Transparent},
+    };
+    // Initialize a background
+    pub fn background_sprite(world: &mut World, sprite_sheet: &SpriteSheetHandle) -> Entity {
+        let mut transform = Transform::default();
+        transform.translation.z = -10.0;
```

Comment:
Magic Numbers. Pull this into a const or config maybe?

Reply:
the background sprite isnt actually in use I think. That was part of the example, I should honestly remove it. I'll do that as a separate commit